docs/getting-started/invite-members/README.md

```diff
@@ -50,7 +51,7 @@ Type the name of the user, placeholder user or group which you want to add. You
 
 Please note that you will have to click on the new member's name or press the Enter key before clicking the Add button.
 
-![add-members](1566224199456.png)
+![searchbar-invite-members](Bildschirmfoto 2022-03-02 um 14.22.44.png)
```

Comment:
please rename the image file and change the name here, so there are no spaces in it.